docs/components/rigid-body.md

```diff
@@ -1,204 +1,230 @@
+| **linvel**               | linear velocity                                                                                               | `x: 0, y: 0, z: 0`             |
+| **angvel**               | angular velocity                                                                                              | `x: 0, y: 0, z: 0`             |
+| **enabledRotations**     | enable rotations in specific axis                                                                             | `{x: true, y: true, z: true }` |
+| **enabledTranslations**  | enable translations in specific axis                                                                          | `{x: true, y: true, z: true }` |
+| **lockTranslations**     | Lock all translations                                                                                         | `false`                        |
+| **lockRotations**        | Lock all rotations                                                                                            | `false`                        |
+| **enableCcd**            | Enable continuous collision detection                                                                         | `false`                        |
+| **friction**             | The friction coefficient of this collider. (automatic-collider)                                               | `0.5`                          |
+| **mass**                 | Mass of the collider. (automatic-collider)                                                                    | `1`                            |
+| **density**              | Restitution controls how elastic (aka. bouncy) a contact is. (automatic-collider)                             | `0`                            |
+| **restitution**          | The collider density. If non-zero the collider's mass and angular inertia will be added. (automatic-collider) | `1`                            |
+| **activeCollision**      | To set the collider receiver/emitter collision events (automatic-collider)                                    | `false`                        |
+| **activeCollisionTypes** | Type of the collision event. (automatic-collider)                                                             | `ActiveCollisionTypes.DEFAULT` |
+| **collisionGroups**      | To specify collision groups. (automatic-collider)                                                             | `undefined`                    |
+
+:::info
+The `rigidBody` instance has many other functions, please check the
+[official docs](https://rapier.rs/docs/api/javascript/JavaScript3D/) for a
+complete list, if you need them, you can
+use[Template ref](https://vuejs.org/guide/essentials/template-refs.html#template-refs).
+:::
+
+## Expose object
+
+```js
+const exposeObject = {
+  instance: rigidBodyInstance,
+  rigidBodyDesc,
+  context: colliderInfos,
```

Comment:
Now I'm thinking to rename it `colliders` instead of `context`

Reply:
Agree